File: Backend API/Locafest/com.locafest.Domain/UsuarioContext/Entities/Cliente.cs
using com.locafest.Domain.UsuarioContext.Enum;
using System;
using System.Collections.Generic;
using System.Text;

namespace com.locafest.Domain.UsuarioContext.Entities
{
    public class Cliente : BaseEntity
    {
        public string Nome { get; set; }
        public string Cpf { get; set; }
        public DateTime Aniversario { get; set; }
        public Guid UsuarioId { get; set; }
        public Guid EnderecoId { get; set; }
        public Usuario Usuario { get; set; }
        public Endereco Endereco { get; set; }
        public int PerfilId { get { return (int)PerfilEnum.Cliente; } }
    }
}

File: Mobile FrontEnd/locafest/lib/app/shared/entities/cliente.dart
class Cliente {
  String id;
  String nome;
  String cpf;
  String aniversario;
  String usuarioId;
  String enderecoId;

  Cliente(
      {this.id,
      this.nome,
      this.cpf,
      this.aniversario,
      this.usuarioId,
      this.enderecoId});

  Cliente.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    nome = json['nome'];
    cpf = json['cpf'];
    aniversario = json['aniversario'];
    usuarioId = json['usuarioId'];
    enderecoId = json['enderecoId'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['id'] = this.id;
    data['nome'] = this.nome;
    data['cpf'] = this.cpf;
    data['aniversario'] = this.aniversario;
    data['usuarioId'] = this.usuarioId;
    data['enderecoId'] = this.enderecoId;
    return data;
  }
}